pom.xml

```
@@ -26,7 +26,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <jackson.version>[2.8.11,2.9.8]</jackson.version>
+        <jackson.version>[2.8.11,2.9.9]</jackson.version>
```

```suggestion
        <jackson.version>[2.8.11,2.10.0)</jackson.version>
```

All 2.9 should work because jackson follow semver rules (https://github.com/FasterXML/jackson/wiki/Jackson-Releases)

---

Agreed.